src/app/examples/badge-example/badge-example.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="badge-examples">
+  <kirby-badge colortype="danger">1</kirby-badge>
+  <kirby-badge colortype="alert">99</kirby-badge>
```

Comment:
Consider adding example with 3 ciphers:
```suggestion
  <kirby-badge colortype="alert">99</kirby-badge>
  <kirby-badge colortype="alert">123</kirby-badge>
```